lib/ExpandableListItem/ExpandableListItem.less

```diff
@@ -27,6 +27,9 @@
 .moon-expandable-list-item-client.indented {
 	padding-left: @moon-item-indent + @moon-spotlight-outset;
 }
+.enyo-locale-right-to-left .moon-expandable-list-item-header.moon-expandable-list-item-header {
```

Comment:
What's the reasoning for doubling-up the selector classes here?
`.moon-expandable-list-item-header.moon-expandable-list-item-header`
Couldn't just one of these 2 classes do the same job?